lxd/network/driver_ovn.go

```diff
@@ -5072,8 +5074,11 @@ func (n *ovn) LoadBalancerUpdate(listenAddress string, req api.NetworkLoadBalanc
 		}
 
 		newLoadBalancer := api.NetworkLoadBalancer{
-			ListenAddress:          curLoadBalancer.ListenAddress,
-			NetworkLoadBalancerPut: req,
```

Comment:
As req is NetworkLoadBalancerPut we should be using api.NetworkLoadBalancer.SetWritable() to avoid duplication.